### PR TITLE
Fix FPS gui

### PR DESCRIPTION
**Pull request by ccuser44**
Currently it only shows the render FPS (and a bit improperly).

Now this specifies both the render and physics FPS which are different
It does not fix the invalid reporting of the render FPS though

### CODE REVIEW

File: MainModule/Client/UI/Default/FPS.lua [hunk 1]
```diff
@@ -41,7 +41,7 @@ return function(data,env)
 		window:Ready()
 
 		repeat
-			label.Text = `{math.floor(1/service["RunService"].RenderStepped:Wait())} fps`
+			label.Text = `Render: {math.floor(1/service["RunService"].RenderStepped:Wait())} fps\nPhysics: {workspace:GetRealPhysicsFPS()} fps`
```
Comment (watameln) [inline MainModule/Client/UI/Default/FPS.lua:44]: please math.floor

Comment (ccuser44) [reply MainModule/Client/UI/Default/FPS.lua]: Done

**Comment by kaiserandaxl** [comment]
@ccuser44 you do realize FPS is how many frames are rendered, not physics.

**Comment by kaiserandaxl** [comment]
You sound dumb right now, FPS is the amount of frames **rendered** in a second, not how fast physics runs?

**Comment by kaiserandaxl** [comment]
And if you want more accurate readings with physics and render just average them out

**Comment by ccuser44** [comment]
> You sound dumb right now, FPS is the amount of frames rendered in a second, not how fast physics runs?

Wrong
FPS means the amount of frames calculated per second
Physics frames are calculated sperately

Im not the one who makes the distinction the Roblox client shows this the same too
Do `Ctrl + Shift + F5`

**Comment by ccuser44** [comment]
Proof of work
image

**Comment by Dimenpsyonal** [comment]
Ah bril

**Pull request merged**